_data/versions.yml

```diff
@@ -19,3 +19,7 @@
       version: master
      flexvol:
       version: master
+     install-operator:
```

Comment:
Soo this is interesting. Due to some complex dependency ordering in the tigera/process hashrelease Makefile, we have been placing Operator versioning information at the toplevel of the release blob, e.g.

```yaml
- title: 2020-01-10-v2-6-worldly
  dockerRepo: gcr.io/unique-caldron-775/cnx
  helmRelease: 0
  calico:
    minor_version: v3.10
  components:
    cnx-node:
      image: tigera/cnx-node
      version: v2.6.1-4-ga51b97e
  tigera-operator:
    image: tigera/operator
    registry: gcr.io/unique-caldron-775/cnx
    version: 2020-01-10-v3-10-hovedset-2020-01-10-v2-6-worldly
```

I would need some time to dig up and refresh on the precise reason why. but for now, It'd be good to stay consistent with that layout.